pages/public-gateways/reference-content/understanding-v2.mdx

```diff
@@ -0,0 +1,207 @@
+
+Whether you choose a custom or default CIDR block, automatic address assignment or use a reserved address, **the resource's private IP address does not risk changing** unless you detach the resource from the Private Network. To ensure that you can keep the same address for a resource even after detaching it, use the [reserve IP](/network/ipam/how-to/reserve-ip/) functionality.
+
+## Introducing Public Gateways API v2
+
+Since the assignment and management of IP addresses on Private Networks is now managed by IPAM and the Private Networks themselves, we must complete the removal of the DHCP functionality from Public Gateways. This means releasing a new version (v2) of the [Public Gateways API](https://www.scaleway.com/en/developers/api/public-gateway/), which has until now retained a number of legacy DHCP functions. From this new version, you can expect:
+
+- IPAM mode becomes default
+- Removal of the DHCP object
+- Removal of the `address` field
+- Removal of DHCP entries
+- New SSH bastion feature: Allowed IPs
+
+Read on to find out more.
+
+### IPAM mode becomes default
+
+Scaleway Public Gateways are either in **Legacy mode** or **IPAM mode**. You can see the mode of a given gateway by:
+
+- Checking its badge in the gateway listing page of the [Scaleway console](https://console.scaleway.com/public-gateway/public-gateways) (see screenshot below).
+- Checking the value of the `is_legacy` field via the [Public Gateways API](https://www.scaleway.com/en/developers/api/public-gateway/#path-gateways-get-a-public-gateway).
+
+<Lightbox src="scaleway-pgw-listing.webp" alt="A screen in the Scaleway console shows a listing of two Public Gateways, one of which has a badge saying IPAM, the other a badge saying Legacy." />
+
+<Message type="tip">
+All Public Gateways created via the Scaleway console since 17 October 2023 are necessarily in IPAM mode.
+</Message>
+
+Legacy Public Gateways use a [workaround](/network/vpc/reference-content/vpc-migration/#public-gateways-and-vpc) to ensure IPAM compatibility. IPAM-mode Public Gateways are fully integrated with Scaleway's [IPAM](/network/ipam/concepts/#ipam), which manages the coherent assignment of IP addresses to the gateway itself, and resources on attached Private Networks.
+
+Legacy mode will be deprecated going forward, and will not be compatible with v2 of the Public Gateway API. It will no longer be possible to create legacy-mode Public Gateways, all gateways will necessarily be in IPAM mode.
```

Comment:
```suggestion
Legacy mode will be deprecated going forward, and will not be compatible with v2 of the Public Gateway API. It will no longer be possible to create legacy-mode Public Gateways, all new gateways will necessarily be in IPAM mode.
```